dev/com.ibm.ws.concurrent.mp.1.0/src/com/ibm/ws/concurrent/mp/ContextualRunnable.java

```diff
@@ -68,6 +68,12 @@ public void run() {
             try {
                 if (contextApplied != null)
                     threadContextDescriptor.taskStopping(contextApplied);
+            } catch (RuntimeException x) {
+                // prioritize surfacing an actual task failure over a failure clearing context
```

Comment:
If we switch the prioritization in ContextualRunnable, we should do so in every other contextual action type as well (Function, BiFunction, Consumer, Supplier, ...).   That said, I'm not convinced that we ought to switch the ordering.  A failure to clear context is very severe, and typically would be more important than a task's own exception (which might even sometimes be expected).  I would argue that also applies in the case for which these changes are being added.